### ISSUE TITLE
PE Open book

### ISSUE THREAD

**Issue by theodorekwok** [nus-cs2113-AY2122S1/forum#139]
Hi, is the PE for this Friday open book? Like are we allowed to have like a physical book to refer to for notes or writing?

**Comment by okkhoy** [comment]
Yes. You can write your own notes during PE (however, be mindful, you are constrained by the time allowed)

**Comment by theodorekwok** [comment]
Ok thanks prof!
